module/zfs/zfs_vfsops.c

```diff
@@ -972,6 +972,152 @@ zfs_id_overquota(zfsvfs_t *zfsvfs, uint64_t usedobj, uint64_t id)
 	    zfs_id_overobjquota(zfsvfs, usedobj, id));
 }
 
+static zfsvfs_kstats_t empty_zfsvfs_kstats = {
```

Comment:
in theory, we could add the dataset name here

Reply:
See my other comment

Reply:
Review update added a field with the dataset name.